### ISSUE TITLE
Reliable consent frame detection during video creation

### ISSUE THREAD

**Issue by Datamance** [lookit/lookit-api#258]
**TL;DR**: 
Have a non-fragile way of detecting whether or not a given video is consent footage or not.

**Narrative**
As a programmer, relying on frame ID to determine whether or not something is consent footage seems fairly brittle.

**Acceptance Criteria**
- [x] New tests for RenameVideoView and/or Video Model pass

**Implementation Notes**
Server: Video model's `from_pipe_payload` method
Client: Send new metadata from one of the video frame components?

*Update 4/2*: per [the commit](https://github.com/lookit/lookit-api/commit/b336d662719b8b367910ad99b9a9e9222f086e6a) addressing #129, we've introduced a `FrameActionDispatcher` class that should make this trivial

**Comment by Datamance** [comment]
This is taken care of through `FrameActionDispatcher`, closing out

**Comment by kimberscott** [comment]
Is this actually used for identifying consent footage now, or does the videoType just exist if we did want to use it?

**Comment by Datamance** [comment]
haha april fools not really but that's what I'll tell myself

I'll modify the acceptance criteria to reflect what's been done so far
